src/main/java/taa/command/assessment/AddAssessmentCommand.java

```diff
@@ -58,7 +58,8 @@ public void checkArgument() throws TaaException {
         if (!Util.isStringDouble(maximumMarksString)) {
```

Comment:
Change to `isStringDouble(maximumMarksString, 2)` to only allow 2dp

Reply:
ok updated